src/inc/corjit.h

```diff
@@ -113,18 +113,22 @@ class CORJIT_FLAGS
         CORJIT_FLAG_USE_AVX                 = 14,
         CORJIT_FLAG_USE_AVX2                = 15,
         CORJIT_FLAG_USE_AVX_512             = 16,
-        CORJIT_FLAG_FEATURE_SIMD            = 17,
 
     #else // !defined(_TARGET_X86_) && !defined(_TARGET_AMD64_)
 
         CORJIT_FLAG_UNUSED6                 = 13,
         CORJIT_FLAG_UNUSED7                 = 14,
         CORJIT_FLAG_UNUSED8                 = 15,
         CORJIT_FLAG_UNUSED9                 = 16,
-        CORJIT_FLAG_UNUSED10                = 17,
 
     #endif // !defined(_TARGET_X86_) && !defined(_TARGET_AMD64_)
 
+    #if defined(_TARGET_X86_) || defined(_TARGET_AMD64_) || defined(_TARGET_ARM64_)
```

Comment:
This could use #ifdef FEATURE_SIMD  here

Reply:
I don't think FEATURE_SIMD is available in all places where corjit.h is included.

Reply:
We should change FEATURE_SIMD into a standard FEATURE tag and define it in jit.h as 0/1

Reply:
Maybe; that's unrelated. corjit.h is included in lots of the VM that does not include jit.h.

Reply:
!! Note that this is a change to the inc directory, which requires a corresponding checking on the Desktop !!